### PR TITLE
Added group `tour`

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1730,12 +1730,391 @@ root will not be listable, as it is virtual. Access known paths directly.
...
+        Content-Type: text/plain
+        Server: go-ipfs/0.4.0-dev
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        X-Stream-Output: 1
+        Date: Thu, 11 Feb 2016 22:48:34 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (string)
```
Comment (dignifiedquire) [inline apiary.apib:1819]: Is this needed?

File: apiary.apib [hunk 2]
```diff
@@ -1730,12 +1730,391 @@ root will not be listable, as it is virtual. Access known paths directly.
...
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        X-Stream-Output: 1
+        Date: Thu, 11 Feb 2016 22:50:39 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (string)
+
+    + Body
```
Comment (dignifiedquire) [inline apiary.apib:2114]: Is this needed?

File: apiary.apib (hunk 1 above)
Comment (RichardLitt) [inline apiary.apib:1819]: The MSON type definition is optional; but I don't see a reason not to add this.

**Comment by RichardLitt** [comment]
RFM?

File: apiary.apib (hunk 2 above)
Comment (RichardLitt) [inline apiary.apib:2114]: I default to including an empty body if there is no response, yes. Seems a better way than not including Body at all.

File: apiary.apib (hunk 1 above)
Comment (dignifiedquire) [inline apiary.apib:1819]: :+1:

**Comment by RichardLitt** [comment]
Thanks, @dignifiedquire!

**Pull request merged**